app/javascript/components/ingredients/ingredients-container.vue

```diff
@@ -193,11 +193,14 @@ export default {
       this.showingAdd = !this.showingAdd;
 
       try {
+        var ingredientsInfo = this.$refs.addIngredientInfo.form;
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>All 'var' declarations must be at the top of the function scope   vars-on-top